var/spack/repos/builtin/packages/eospac/package.py

```diff
@@ -15,7 +15,9 @@ class Eospac(Package):
     homepage = "http://laws.lanl.gov/projects/data/eos.html"
     list_url = "http://laws.lanl.gov/projects/data/eos/eospacReleases.php"
 
-    version('6.4.1',       sha256='2310c49bd7a60cad41d2cb1059c5f0a1904f0c778b164937182382df326ca003', preferred=True,
+    version('6.4.2beta',   sha256='635b94f1ec7558deca92a3858c92db0f4437170252bb114cbdb809b74b6ee870', preferred=True,
```

Comment:
We don't want to install a beta version by default, you should keep `preferred=True` on the last stable release.

Reply:
It's unfortunate that EOSPAC's maintainers use the label "beta" to describe some versions, but the difference between 6.4.1 and 6.4.2beta is only one bug fix, for a bug we reported. I'm happy to move `preferred=True` back to the last "stable" release, but no one should actually prefer that previous version.

Reply:
In that case, you can make your new version preferred. It might be useful to add a comment to the package saying that beta doesn't mean beta.

Reply:
Sounds good, thanks. I've added a comment above the list of versions; let me know if that's still too muddy.